### DISCUSSION TITLE
Lora Issues

### DISCUSSION THREAD

**Discussion by vladmandic** [vladmandic/sdnext#186] [General]
If you're having issues loading Lora and account errors such as 

> RuntimeError: output with shape [128, 320, 1, 1] doesn't match the broadcast shape [128, 320, 3, 3]

That is most likely either not pure **Lora** (it can be actual **Locon** or **LyCORIS** while still being marked on sites such as Civit.AI as Lora) or its **Lora** produced with latest version of `Kohya_ss` code which adds additional complexities that original A1111 Lora network cannot understand.

Advice is instead to use LyCORIS network instead and attempt to load such **Lora** using `<lyco:lora-name:strength>` instead of `<lora:lora-name:strength>`

**Comment by myndxero** [comment]
Any bearing here on this error? I don't know why loras broke, but they were fine a few days ago until I disabled them and then re-enabled today for first time since then.

![image](https://user-images.githubusercontent.com/113237671/234354294-41ff6f83-580e-43be-891c-cc2339b75b6f.png)

**Comment by vladmandic** [comment]
@myndxero thats because you've installed an extension that is known to cause problems. see #225

**Comment by mart-hill** [comment]
Is it also for the same reason that expression <lora:name:weight> in the prompt doesn't work anymore? 
By the way, could the parsing of it be "redirected" to <lyco:name:weight>, so "older" prompts would still be interpreted? Or is it relatively minor issue? I'm just curious, since it's not a huge inconvenience. 🙂

**Reply by vladmandic** [reply]
`<lora:name:weight>` works and uses specific code path which is old original a1111 implementation of lora. so it wont be able to interpret new loras
`<lyco:name:weight>` uses new code and is much more versatile. can i use this to completely replace old lora code? yes, but its not trivial and i don't have time as long as the issue backlog keeps growing.

**Comment by mart-hill** [comment]
Thank you for claryfiying! It means then, that even the oldest LoRAs I have, seem to be on the newer "side", and it's just that. 🙂

**Comment by myndxero** [comment]
Same issue with and without compostable. I installed composable as a
possible solution at first. Tho honestly thought I had it before, was it
built in at one time?

**Comment by mart-hill** [comment]
I think Composable LoRA extension works with Additional Networks extension, but I'll check that.
